bibliopixel/image.py

```diff
@@ -75,7 +75,7 @@ def loadImage(led, imagePath="", imageObj=None, offset=(0, 0), bgcolor=colors.Of
 
     def setter(x, y, pixel):
         if y >= 0 and x >= 0:
-            texture[y][x] = (r, g, b)
+            texture[y][x] = pixel
```

Comment:
yes... you did, but the problem was that setter didn't exist before and that code was moved from a loop that *had*  r, g, and b defined. But you changed it to just pixel in setter, so it needed update.